### DISCUSSION TITLE
Must run as admin on Windows?

### DISCUSSION THREAD

**Discussion by MarkusPiotrowski** [kevincar/bless#106] [Q&A]
I'm running bless on Windows 10 (64bit), Python 3.8.10, driving a simple notification server. At the moment I do `server.run()` I receive a `WinError 5` Access denied error. This error usually indicates missing permissions. So I need to run my script as administrator (e.g. starting it from a console that was opened in administrator mode). Then it works nicely.

Is this a known issue? It would be nice if one could prevent running the program as admin.

**Comment by kevincar** [comment]
I recognize this may be an old question, but I'm just catching up on maintenance for this repository. 

Is this still an issue you're experiencing? On the number of Windows machines I've tested this on (3+), I've not yet run across this issue. This makes me wonder if your security has more strict permissions over you bluetooth module in general.

**Comment by MarkusPiotrowski** [comment]
Actually, I don't know. I'm still using bless in one of my programs, but I didn't change the code for a long time and possibly I granted the program some permissions that I'm not longer aware of.
